yaml/com/ibm/VPD/Collection.interface.yaml

```diff
@@ -0,0 +1,26 @@
+description: >
+    Implement to provide VPD collection state management.
+
+properties:
+    - name: CollectionStatus
+      type: enum[self.Status]
+      description: >
+          Property which tells the current status of FRU VPD collection.
```

Comment:
is it should be just "VPD collection" or "FRU's VPD collection" ?

Reply:
it gives same meaning.

Reply:
I thought this is for whole VPD collection status of system